java/java.hints/test/unit/src/org/netbeans/modules/java/hints/jdk/ConvertToRecordPatternTest.java

```diff
@@ -92,7 +89,7 @@ public void testDuplicateVarName() throws Exception {
                         + "record Person(String name, int s){}\n"
                         + "public class Test {\n"
                         + "    private int test(Object s) {\n"
-                        + "        if (s instanceof Person(String name, int s1) p) {\n"
+                        + "        if (s instanceof Person(String name, int s1)) {\n"
```

Comment:
@lahodaj i think the tests were wrong? This doesn't seem to be a valid syntax when it also includes the field name.

Another side effect of this PR is that more tests are now active.